src/utils.ts

```diff
@@ -61,3 +61,8 @@ export function combineStylesMemo<T extends Styles>(
     ...style1,
   }));
 }
+
+export const clamp = (value: number, min = 0, max = 100) =>
```

Comment:
what makes 0 - 100 a good default?
if you read somewhere `clamp(x)` do you immediately know that its clamping the value between 0 and 100?
I'd prefer `clamp(x, 0, 100)` tbh.